src/Crowdin.Api/SourceStrings/SourceStringsApiExecutor.cs

```diff
@@ -126,6 +126,30 @@ public async Task<SourceString> EditString(int projectId, int stringId, IEnumera
             CrowdinApiResult result = await _apiClient.SendPatchRequest(url, patches);
             return _jsonParser.ParseResponseObject<SourceString>(result.JsonObject);
         }
+        
+        /// <summary>
+        /// Upload strings status. Documentation:
+        /// <a href="https://developer.crowdin.com/api/v2/string-based/#operation/api.projects.strings.uploads.get">Crowdin API</a>
```

Comment:
There is also Enterprise SB doc - https://developer.crowdin.com/enterprise/api/v2/string-based/